index.html

```diff
@@ -45,35 +45,261 @@
+          id="add-player-button"
+          disabled
+        >
+          Add Player
+        </button>
+      </div>
+      <span class="text-danger" id="duplicate-name" style="display: none"
+        >You can't add duplicate names!</span
+      >
+      <hr />
+      <div id="dice-div" style="display: none">
+        <h5 class="text-center">--- Choose Number of Dices ---</h5>
+        <div class="input-group mb-3">
+          <input
+            type="number"
+            min="1"
+            max="10"
+            class="form-control border border-dark"
+            placeholder="Enter the number of dices to be rolled this round"
+            aria-label="Number of dices to be rolled"
+            aria-describedby="start-round-button"
+            id="no-of-dices"
+            onkeyup="checkNoOfDicesNotEmpty()"
+            onmouseup="checkNoOfDicesNotEmpty()"
+            value="1"
+          />
+          <button
+            class="btn btn-dark border border-dark"
+            type="button"
+            id="start-round-button"
+            disabled
+          >
+            Start Running Round!
+          </button>
+          <button
+            class="btn btn-dark border border-dark"
+            type="button"
+            id="start-knockout-round-button"
+            style="display: none"
+            disabled
+          >
+            Start Knockout Round!
+          </button>
+        </div>
+        <div class="row">
+          <div class="col">
+            <button
+              class="btn btn-secondary btn-light"
+              type="button"
+              id="change-mode-button"
+              style="width: 100%"
+            >
+              Toggle <span class="text-danger">Highest</span>/<span
+                class="text-success"
+                >Lowest</span
+              >
+              Mode
+            </button>
+          </div>
+          <div class="col">
+            <button
+              class="btn btn-secondary btn-light"
+              type="button"
+              id="change-running-knockout-mode-button"
+              style="width: 100%"
+            >
+              Toggle <span class="text-danger">Running</span>/<span
+                class="text-success"
+                >Knockout</span
+              >
+              Mode
+            </button>
+          </div>
+        </div>
+        <hr />
+      </div>
+      <div id="running-history-div" style="display: none">
+        <h5 class="text-center">--- History [Running Mode] ---</h5>
+        <div id="running-history-table"></div>
+      </div>
+      <div id="knockout-history-div" style="display: none">
+        <h5 class="text-center">--- History [Knockout Mode] ---</h5>
+        <div id="knockout-history-table"></div>
+        <hr />
+      </div>
     </div>
     <!-- Import program logic -->
     <script src="script.js"></script>
+    <script src="knockoutscript.js"></script>
     <!-- Define button click functionality -->
     <script>
```

Comment:
Good job using DOM! consider putting this section into a separate .js file instead